app/src/main/java/org/coepi/android/ui/alerts/AlertsViewModel.kt

```diff
@@ -57,6 +61,12 @@ class AlertsViewModel(
         navigation.navigate(ToDirections(actionGlobalAlertsDetails(Args(alert.alert))))
     }
 
+    fun onAlertDismissed() {
+        val notificationManager: NotificationManager =
+            context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+        notificationManager.cancel(LocalNotificationChannelId.INFECTION_REPORTS_CHANNEL.ordinal)
```

Comment:
Don't access system services directly in view models. Put this in a dedicated class that's injected. We also have already some classes to manage notifications. Maybe it can be added to them.